src/com/google/enterprise/adaptor/DocIdSender.java

```diff
@@ -211,16 +216,70 @@ public DocId pushNamedResources(Map<DocId, Acl> resources,
     return null;
   }
 
+  @Override
   public GroupPrincipal pushGroupDefinitions(
       Map<GroupPrincipal, ? extends Collection<Principal>> defs,
-      boolean caseSensitive, ExceptionHandler handler) 
-      throws InterruptedException {
+      boolean caseSensitive, FeedType feedType, String groupSource,
+      ExceptionHandler handler) throws InterruptedException {
     if (config.markAllDocsAsPublic()) {
       log.finest("Ignoring attempt to send groups to the GSA because "
                  + "markAllDocsAsPublic is true.");
       return null;
     }
-    return pushGroupDefinitionsInternal(defs, caseSensitive, handler);
+    return pushGroupDefinitionsInternal(
+        defs, caseSensitive, feedType, groupSource, handler);
+  }
+
+  /**
+   * Issue: Full vs. Incremental Feeds for Groups.
```

Comment:
Would it be more accurate to say the issue is "When full group pushes  exceed permissible size of single feed".

The issue's descroption seems right. The label of the issue seems not right.